.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "jekyllbook"]
+	path = jekyllbook
+	url = https://github.com/ebetica/jekyllbook
```

Comment:
Oh? Shall I fork it and point to my version (for maintainability)?

Reply:
If you want? I can put you as a contributor as well if you like ;p

Reply:
This website will break if your repo cease to exist.